README.md

```diff
@@ -194,6 +194,32 @@ brew install docker-completion
 Now you will need to start the Docker service: run Spotlight and type in
 'docker'.
 
+### Setup: PostgreSQL
+
+To run postgres in Docker, first you will need the image:
+
+```console
+docker pull postgres:11.6
+```
+
+Now, you can run postgres locally:
+
+```console
+$ docker run --detach --publish 5432:5432 postgres:11.6
+8a6196a9ae85286e3598bc49a1a59954a3762b633059829389af333964041215
+$ psql --username postgres --host localhost
```

Comment:
As we talked about, we should add some documentation on how to intall the postgres client (and whether we're using psql or pgcli).

Reply:
How's that?

Reply:
Not to be too pedantic, but instead of "(available in Homebrew)", can you include `brew install pgcli` in a small code snippet? The README follows that pattern for the other dependencies.